lib/src/components/NxCollapsibleItems/NxCollapsibleItems.scss

```diff
@@ -228,4 +228,11 @@ button, a {
       }
     }
   }
+
+  .nx-collapsible-items__trigger {
+    &:hover {
```

Comment:
When they are disabled in a sidebar they're still getting hover states

Reply:
> When they are disabled in a sidebar they're still getting hover states

Updated: 3ceb744b03a24929535c47630e9a74217b755fbd